### PR TITLE
Update tests syntax to new flat syntax

### PR DESCRIPTION
**Pull request by citisolo**
more tests

### CODE REVIEW

File: test/interp/nested-if.txt [hunk 1]
```diff
@@ -7,14 +7,18 @@
...
-        (i32.const 1)
-        (drop (i32.const 2))
-        (if                   ;; inner if
-          (i32.const 3)
-          (br $exit))))
-    (i32.const 4)))
+    block $exit
+      i32.const 1
+      if 
+        i32.const 2           ;; outer if
```
Comment (binji) [inline test/interp/nested-if.txt]: move comment up to if

**Comment by binji** [comment]
lgtm. By the way, next time you push a change, can you update it so it doesn't include the previous commits? It would make reviewing a bit easier. Thanks!

**Comment by citisolo** [comment]
Sorry I didn't realize I was doing that.  How would I update it , what I've been doing is just pushing to the branch?

**Comment by binji** [comment]
I think you just need to update your branch to `wabt:master`.

**Pull request merged**